srcpkgs/budgie-desktop/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'budgie-desktop'
+pkgname=budgie-desktop
+version=10.5.1
+revision=1
+archs="i686 x86_64*"
```

Comment:
please change this to `i686*`

Reply:
looks ok otherwise.

Reply:
Looks like it got merged before I pushed the changes.

Reply:
Is this actually a issue with only supporting specific architectures or just cross compiling issues?

Reply:
It cross compiled fine for aarch64 and armv7l...

Reply:
we can enable cross compilation right now.

Reply:
> Is this actually a issue with only supporting specific architectures or just cross compiling issues?

I thought it was the former, which it seems isn't true. It cross compiles fine locally as Duncaen said.
So I should remove `archs` in next revision?